luaui/Widgets/cmd_bar_hotkeys.lua

```diff
@@ -312,16 +312,56 @@ local function makeBindsTable(keyLayout)
 		{      "8", "specteam", "7"  },
 		{      "9", "specteam", "8"  },
 
-		{ "Any+0", "group0" },
-		{ "Any+1", "group1" },
-		{ "Any+2", "group2" },
-		{ "Any+3", "group3" },
-		{ "Any+4", "group4" },
-		{ "Any+5", "group5" },
-		{ "Any+6", "group6" },
-		{ "Any+7", "group7" },
-		{ "Any+8", "group8" },
-		{ "Any+9", "group9" },
+{            "0", "group", "0"            },
```

Comment:
The action names are a bit different from the new ones, I made the list to replace with here: https://gist.github.com/badosu/d7b4eae890f4b2888a7cb35ecc454dc1

Use this list and we're good to merge!